### PR TITLE
Perf tweaks

### PR DESCRIPTION
**Pull request by tbkka**
@allevato:  What do you think of these two changes?

### CODE REVIEW

File: Performance/perf_runner.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ readonly script_dir="$(dirname $0)"
 
 # Change this if your checkout of github.com/google/protobuf is in a different
 # location.
-readonly GOOGLE_PROTOBUF_CHECKOUT="$script_dir/../../protobuf"
+readonly GOOGLE_PROTOBUF_CHECKOUT=${GOOGLE_PROTOBUF_CHECKOUT:-"/Users/tkientzle/projects/protobuf"}
```
Comment (allevato) [inline Performance/perf_runner.sh]: Well, this one only applies to your system ;)

File: Performance/perf_runner.sh [hunk 2]
```diff
@@ -106,10 +106,14 @@ function run_harness_and_concatenate_results() {
     "$language": {
 EOF
 
+  echo "Running $language test harness alone..."
+  sleep 3
+  DYLD_LIBRARY_PATH="$script_dir/_generated" "$harness" "$partial_results"
```
Comment (allevato) [inline Performance/perf_runner.sh]: +1, this is probably the right thing to do. It likely won't affect the comparative nature of the timings, but it'll give us truer absolute numbers.

File: Performance/perf_runner.sh (hunk 1 above)
Comment (tbkka) [inline Performance/perf_runner.sh]: Oops.  Fixed the default here.

**Comment by tbkka** [comment]
I didn't really look at the Instruments overhead.  What I noticed rather was that the variability was really high.  Before this, for example, I might see most samples were 0.1s but one would be ~15s.  After this, I still see some outliers but nothing of quite that magnitude any more.

**Pull request merged**